compiler/luci-interpreter/src/Interpreter.cpp

```diff
@@ -174,8 +195,21 @@ void Interpreter::readOutputTensor(const luci::CircleOutput *output_node, void *
 
 void Interpreter::interpret()
 {
-  for (const auto &kernel : _execution_sequence)
+  for (const loco::Node *loco_node :
+       loco::postorder_traversal(loco::output_nodes(const_cast<loco::Graph *>(_main_graph))))
   {
+    const auto *node = dynamic_cast<const luci::CircleNode *>(loco_node);
```

Comment:
It would be better to use `loco::must_cast` to avoid complains from our static analysis tool.

Reply:
Fixed, thank you